### DISCUSSION TITLE
mapper name leading to different profiles

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#17234] [Q&A] [answered]
when clicking on the mapper name In song select, it leads to some other profile, and when clicking on the name in the 'details' page, it leads to the right one.

map: https://osu.ppy.sh/beatmapsets/1544463#osu/3191101

https://user-images.githubusercontent.com/78441708/158079368-78a7ebea-5983-4a18-ba11-42a155672929.mp4

[logs.zip](https://github.com/ppy/osu/files/8240611/logs.zip)

osu!lazer 2022.306.0 
running arch linux

**Comment by frenzibyte** [comment, marked as answer]
The "wrong" mapper is actually a guest mapper of the difficulty ["Yuuya's Hard"](https://osu.ppy.sh/beatmapsets/1544463/discussion/3191101/timeline), guest mappers are not handled properly in osu!(lazer) yet. Duplicate of https://github.com/ppy/osu/issues/12965
